### ISSUE TITLE
Dell EMC S5248 - Fail to bring up link with 10g or 40g transceivers 

### ISSUE THREAD

**Issue by waliulislam** [open-switch/opx-platform-config#71]
Link does not come up when I have 10 gig transceiver on 25 gig front end port or 40 gig transceiver on 100 gig port. The peer device is S6010. On 10gig I have tried both DAC and fiber cable. 40 gig port I have tried DAC cable.

 ```
root@OPX:~# opx-ethtool e101-018-0
Settings for e101-018-0:
dell-pg/port-groups-state/port-group-state/id = port-group-5
dell-pg/port-groups/port-group/id = port-group-5
    Channel ID:   0
    Transceiver Status: Enable
    Media Type: SFP+ 10GBASE-CR-3.0M
    Part Number: 616760003
    Serial Number: CN027GG53711BV0
    Qualified: Yes
    Administrative State: UP
    Operational State: DOWN
    Supported Speed (in Mbps):  [1000, 10000, 25000]
    Auto Negotiation : off
    Configured Speed   : 0
    Operating Speed   : 0
    Duplex   : full
root@OPX:~# 
```

**Comment by jeff-yin** [comment]
@Tejaswi-Goel to determine short-term solution of the set of CPS SET commands needed to bring the interfaces in the port-group up.
@nchervela to enhance `opx-config-fanout` to account for port-group configuration.

**Comment by Tejaswi-Goel** [comment]
Providing the short-term solution for the fanout and fanin commnd to work on 52xx platform.

As discussed with the development team member, s52xx platforms require an extra step of doing fanout for the transceiver speed to be detected on the required port.

**Comment by Tejaswi-Goel** [comment]
**Issue fixed with the changes in the following files:**
**nas_os_if_utils.py** : add changes to find out which port-group the given port is part of and get the current breakout-mode, supported breakout-modes and speed configured on the port-group.
**opx-config-fanout**: changes to get the mode and speed configuration from the port group the given interface is part of .
**nas_yang_values.py**: changes to add support for 4x4 breakout-mode as sfp28 ports only support 4x4 breakout mode.

Final utility to published as part of #72 bug.

**Comment by waliulislam** [comment]
Just having a 10g transceiver will not set the port to 10g. We have to fanout the port to 10g using new opx-config-fanout command that comes with 3.1.0-rc0
